### PR TITLE
Add anchor links to headers

### LINKED ISSUE #1075: No anchor elements to get links to headers on about pages
On the about pages like <https://docs.rs/about/builds> there's no anchor elements on the headers to get direct links to them like <https://docs.rs/about/builds#detecting-docsrs>.

### PR DESCRIPTION
**Pull request by segfult**
resolves #1075

### CODE REVIEW

File: templates/core/about/builds.html [hunk 1]
```diff
@@ -19,15 +19,21 @@ <h1>Builds</h1>
         {%- endif -%}
     </p>
 
-    <h3 id="notes-on-docsrs">Notes on using Docs.rs</h3>
+    <h3 id="notes-on-docsrs">
+			<a href="#notes-on-docsrs">Notes on using Docs.rs</a>
+		</h3>
```
Comment (jyn514) [inline templates/core/about/builds.html:24]: Do you mind using spaces to match the rest of the file?

**Review by jyn514 — COMMENTED** [review]
Can you a screenshot or video of what this looks like?

**Comment by segfult** [comment]
> Can you post a screenshot or video of what this looks like?

Hey @jyn514, it looks like this.
gg

**Comment by jyn514** [comment]
Thanks! That looks good :) do you mind adding the same links to the other /about pages? In particular the main `/about` page could use links I think.

**Comment by segfult** [comment]
> Thanks! That looks good :) do you mind adding the same links to the other /about pages? In particular the main `/about` page could use links I think.

Sure, no problem :).
>In particular the main `/about` page could use links I think.

That would be this right? https://github.com/rust-lang/docs.rs/blob/master/templates/core/about/index.html

**Comment by jyn514** [comment]
Yes, that's the right file.

File: templates/core/about/index.html [hunk 2]
```diff
@@ -25,18 +25,18 @@ <h1 id="crate-title">About Docs.rs</h1>
         </p>
 
 
-        <h2 id="more-about-docsrs">More about Docs.rs</h2>
+	 <h2 id="more-about-docsrs"> <a href="#more-about-docsrs">More about Docs.rs</a> </h2>
```
Comment (jyn514) [inline templates/core/about/index.html:28]: I don't think you meant to change the spacing here, please keep the existing spaces (not tabs).
```suggestion
        <h2 id="more-about-docsrs"> <a href="#more-about-docsrs">More about Docs.rs</a> </h2>
```

File: templates/core/about/index.html [hunk 3]
```diff
@@ -25,18 +25,18 @@ <h1 id="crate-title">About Docs.rs</h1>
...
         <ol>
             <li><a href="/about/badges">Badges</a>: How to use badges generated by Docs.rs</li>
             <li><a href="/about/builds">Builds</a>: How Docs.rs builds documentation for a crate</li>
             <li><a href="/about/metadata">Metadata</a>: How you can configure a build</li>
             <li><a href="/about/redirections">Redirections</a>: How Docs.rs uses semantic versioning in URLs</li>
         </ol>
 
-        <h3 id="version">Version</h3>
+	 <h3 id="version"> <a href="#version">Version</a> </h3>
```
Comment (jyn514) [inline templates/core/about/index.html:36]: Same here.

File: templates/core/about/index.html [hunk 4]
```diff
@@ -25,18 +25,18 @@ <h1 id="crate-title">About Docs.rs</h1>
...
         <p>Currently running Docs.rs version is: <strong>{{ docsrs_version() }}</strong></p>
 
-        <h3 id="contact">Contact</h3>
+	 <h3 id="contact"> <a href="#contact">Contact</a> </h3>
```
Comment (jyn514) [inline templates/core/about/index.html:39]: Same here.

**Comment by segfult** [comment]
@jyn514 My bad, i was using tabs. Fixed it now :)

**Comment by jyn514** [comment]
Looks great, thanks!

**Pull request merged**